docs/types/value-types.rst

```diff
@@ -122,6 +122,12 @@ you are using are large enough to hold the result and prepare for potential wrap
 .. note::
   Note that ``0**0`` is defined by the EVM as ``1``.
 
+.. warning::
```

Comment:
If you put it here, it will be part of the "Exponentiation" section. I think it should go above the "Comparisons" section.

Reply:
@chriseth The reasoning here was that it applies to

> If the result of some operation on those numbers does not fit inside this range, it is truncated

which is all of the above as far as I can tell, it could come after the paragraphs in the `Integer` section, but I kind of wanted it to come after people had seen the operations on those integers.

Reply:
still, logically it will be inside the exponentiation section and not after it. I think it is fine to mention this as early as possible.